database/README.md

```diff
@@ -0,0 +1,41 @@
+## About
```

Comment:
**To consider:**
I would rename the module to `test_database` or `demo_database`.

Reply:
Okay, agreed, will do that